Misc/NEWS.d/next/Library/2019-08-21-16-38-56.bpo-9938.t3G7N9.rst

```diff
@@ -0,0 +1 @@
+Add ``exit_on_error`` in :class:`ArgumentParser`.
```

Comment:
3) Change the Misc/NEWS entry to specify that ``allow_abbrev`` is an optional keyword argument.

```suggestion
Add optional keyword argument ``exit_on_error`` for :class:`ArgumentParser`.
```

Reply:
your desc looks more exact, thanks.